src/components/TodoListItem.module.css

```diff
@@ -0,0 +1,125 @@
+@import url("https://fonts.googleapis.com/css2?family=Zen+Kaku+Gothic+Antique:wght@300;400&display=swap");
+
+:root{
+  --deep-orange:#F7C26C;
```

Comment:
You can use LessCSS instead to manage variable names. It's pretty commonly used in the industry https://lesscss.org/